services/common/src/components/projectSummary/AuthorizationsInvolved.tsx

```diff
@@ -49,46 +49,43 @@ import DocumentTable from "@mds/common/components/documents/DocumentTable";
 import { MineDocument } from "@mds/common/models/documents/document";
 import { Link } from "react-router-dom";
 
-export interface ProjectSummary {
-  project_summary_id: number;
-  mine_guid: string;
-  project_summary_guid: string;
-  status_code: string;
-  submission_date: string;
-  project_summary_description: string;
-  project_guid: string;
-  documents: IProjectSummaryDocument[];
-}
-
-const RenderEMAPermitCommonSections = ({ props }) => {
-  const { code } = props;
+const RenderEMAPermitCommonSections = ({ code, isAmendment, index }) => {
   const dispatch = useDispatch();
-  const purposeLabel = props.isAmendment
+  const purposeLabel = isAmendment
     ? "Additional Amendment Request Information"
     : "Purpose of Application";
 
-  const [showDocSection, setShowDocSection] = useState(props?.exemption_requested || false);
+  const { authorizations, mine_guid, project_guid, project_summary_guid } = useSelector(
+    getFormValues(FORM.ADD_EDIT_PROJECT_SUMMARY)
+  );
+
+  const codeAuthorizations = authorizations[code];
+  const { AMENDMENT, NEW } = codeAuthorizations;
+  const sectionValues = isAmendment ? AMENDMENT[index] : NEW[index];
+  const dfaRequired =
+    isAmendment &&
+    sectionValues?.amendment_changes?.some((val) => ["ILT", "IGT", "DDL"].includes(val));
+  const [showDocSection, setShowDocSection] = useState(sectionValues?.exemption_requested || false);
 
   const projectSummaryDocumentTypesHash = useSelector(getProjectSummaryDocumentTypesHash);
 
   const onChange = (value, _newVal, _prevVal, _fieldName) => {
     setShowDocSection(value);
   };
 
-  const updateAmendmentDocuments = (doc: IProjectSummaryDocument, code: string) => {
-    const index = 0;
-    const response = dispatch(
+  const updateAmendmentDocuments = (doc: IProjectSummaryDocument) => {
+    const type = isAmendment ? "AMENDMENT" : "NEW";
+    dispatch(
       arrayPush(
         FORM.ADD_EDIT_PROJECT_SUMMARY,
-        `authorizations.[${code}].AMENDMENT.[${index}].amendment_documents`,
+        `authorizations.[${code}].${type}[${index}].amendment_documents`,
```

Comment:
this here was for the documents duplicating. The change from AMENDMENT to ${type} is to stop them duplicating across new/amendment, and the index now being a prop set by the parent is what stops them from being duplicated across multiple amendments.